src/containers/List/ListFilters/OptionFilters.tsx

```diff
@@ -75,15 +76,15 @@ const SearchableListFilter: React.FC<SearchableListFilterProps> = ({
   onRemove,
 }) => {
   const { t } = useLanguageProvider()
-  const [searchValue, setSearchValue] = useState('')
+  const [searchValue, setSearchValue] = useDebounce('')
 
   const { query, resultExtractor, variables } = getFilterListQuery({
     searchValue,
     filterListParameters,
   })
 
   const { data, error } = useQuery(query, {
-    fetchPolicy: 'network-only',
+    fetchPolicy: 'cache-first',
```

Comment:
List filters use `cache-first` so they don't make a query at all (which would compete with the main list query). We may have to keep an eye on this. User/Reviewer lists don't change often, but we might need to explicitly invalidate this cache value at certain times.